lib/msal-common/src/authority/ProtocolMode.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Protocol modes supported by MSAL.
+ */
+export enum ProtocolMode {
+    AAD,
```

Comment:
Making these strings is probably a good idea.

Reply:
And b2c should not be included.

Reply:
Changed to strings now.